.github/workflows/push-pr-lint.yaml

```diff
@@ -5,22 +5,23 @@ jobs:
   lint-test:
     runs-on: ubuntu-latest
     steps:
-    - name: Checkout code
-      uses: actions/checkout@v4
+      - name: Checkout code
+        uses: actions/checkout@v4
 
-    - name: Install Go
-      uses: actions/setup-go@v4
-      with:
-        go-version-file: go.mod
+      - name: Install Go
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: go.mod
```

Comment:
oh we need to do this everywhere. Many projects still have a static version to check against.